cmd/fitgen/internal/profile/errors.go

```diff
@@ -0,0 +1,13 @@
+package profile
```

Comment:
We don't need to introduce these types until we need to do anything specific based on the type. Currently the only user, `cmd/fitgen`. does not.

Wrapping. `%w`, could potentially stay.